cycamore_input/predicting_the_past.xml

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" standalone="no" ?>
+
+<simulation xmlns:xi="http://www.w3.org/2001/XInclude">
```

Comment:
> I realized that I will be needing a script to add the reactors to the Cyclus input file other than those produced previously in order to make it work with DeployInst.

Now that I think about it, I think this would actually work better by just using jinja to include the reactors in the facility tag and their names in the deploy tag. This way, the current predicting_the_past.xml file would be just another template.

In doing this, I will edit the recipy_generator.py script, so that the user just needs to run one script where they can specify the reactor name that they want to add to the simulation and this would produce a cyclus input xml file.